CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ to [sourceforge feature requests](https://sourceforge.net/p/jabref/features/) by
 
 ### Changed
 - All import/open database warnings are now shown in a scrolling text area
+- Implemented #647 (partly): The preview can now be copied
```

Comment:
Since Olifair in #647 in content with the `Copy of preview` (and the other suggestions are now tracked at #119), I would just remove the "partly" and close #647 after merging this PR.